qunit-dom-codemod.js

```diff
@@ -45,6 +45,33 @@ export default function(file, api, options) {
     return j(`x({ count: ${count} })`).find(j.ObjectExpression).paths()[0].node;
   }
 
+  // assert.equal(anything, true) -> assert.ok(anything);
+  // assert.equal(anything, false) -> assert.notOk(anything);
+
+  root.find(j.CallExpression, assertEqual)
+    .filter(p => j.match(p.get('arguments').get('1'), {
+        type: 'Literal',
+        value: v => v === true || v === false
+      }
+    ))
+    .forEach(p => {
+      let args = [p.node.arguments[0]];
+      if (p.node.arguments.length > 2) {
+        args.push(p.node.arguments[2]);
+      }
+
+      p.replace(
+        j.callExpression(
+          j.memberExpression(
+            j.identifier('assert'),
+            j.identifier(p.node.arguments[1].value === true ? 'ok' : 'notOk')
+          ),
+          args
+        )
+      );
+
+    });
```

Comment:
hmmmm.... this replaces all of those assertions though, not just the ones that are later replaced by `qunit-dom` assertions, right?

Reply:
True. Don't you think that makes sense?

Reply:
in general yes, but it might be surprising for people that the `qunit-dom` codemod changes assertions that are not related to `qunit-dom` at all 🤔 

I'd prefer it if we could scope it to just the assertions that are actually relevant, but if that significantly increases the code complexity it might not be worth it either...

Reply:
@Turbo87 Done! Wasn't that bad...